src/main/java/spring/practice/dao/impl/UserDaoImpl.java

```diff
@@ -49,4 +49,16 @@ public void add(User user) {
             throw new RuntimeException("Can't get list of Users", e);
         }
     }
+
+    @Override
+    public User get(Long id) {
+        try (Session session = sessionFactory.openSession()) {
+            Query<User> getUserByIdQuery = session.createQuery("FROM User u "
```

Comment:
https://mate-academy.github.io/jv-program-fulltime/04_spring/checklist/hw_03.html

> Don’t complicate get by id methods in your DAO, use session.get().

Reply:
refactored due to changing `get` method